hazelcast/src/main/java/com/hazelcast/sql/impl/schema/map/PartitionedMapTableResolver.java

```diff
@@ -112,18 +111,18 @@ private PartitionedMapTable createTable(
 
             boolean hd = mapContainer.getMapConfig().getInMemoryFormat() == InMemoryFormat.NATIVE;
```

Comment:
`PartitionedMapTable#hd` is not really used anymore. Does it make sense to keep it?

Reply:
I'd keep it for now, but will clean it up in the next PRs according IMDG engine removal.